lib/fastlane/plugin/revenuecat_internal/actions/replace_version_number_action.rb

```diff
@@ -36,15 +36,22 @@ def self.available_options
                                        type: String),
           FastlaneCore::ConfigItem.new(key: :files_to_update,
                                        env_name: "RC_INTERNAL_FILES_TO_UPDATE_VERSION",
-                                       description: "Files that contain the version number and need to have it updated",
+                                       description: 'Hash of files that contain the version number and need to have it' \
+                                                    'updated to the patterns that contains the version in the file.' \
+                                                    'Mark the version in the pattern using {x}.' \
+                                                    'For example: { "./pubspec.yaml" => ["version: {x}"] }',
                                        optional: false,
-                                       type: Array),
+                                       type: Hash),
```

Comment:
I was trying to look in the documentation, is this an accepted type? I couldn't find it in the documentation... The closes I could find was polymorphic arguments: https://docs.fastlane.tools/advanced/actions/

But well, if you've tested it and it works, it should be ok.

Reply:
I couldn't find it either! But I saw it being used in their repo https://github.com/search?q=repo:fastlane/fastlane%20type:%20Hash&type=code So I assumed they just hadn't documented it